server/src/lsp-server.spec.ts

```diff
@@ -491,106 +491,58 @@ describe('code actions', () => {
 
         assert.deepEqual(result, [
```

Comment:
TS 4 has different quick-actions. Note that this won't change if someone's running the language server on an older typescript version, since these are pulled from the tslib.